src/index.js

```diff
@@ -1,12 +1,28 @@
+/* eslint-disable import/extensions */
```

Comment:
- [ ] Kindly do not disable linters like the previous reviewer mentioned.

Reply:
Hello @Ghiftee 
I write for PR and we put as solution that. the problem is  this I resolve the issues manually but working fine on locally but on GitHub not pass a the eslint when i see the error inside action was missing extinction. thus when i add the extension, execute error again so that why i use /* eslint-disable-next-line */ i can find any solution if you have any recommend solution I will can fix again Thank you.